doc/topics/releases/releasecandidate.rst

```diff
@@ -12,10 +12,15 @@ features </topics/releases/2016.3.0>`! Be sure to report any bugs you find on `G
 Installing Using Packages
 =========================
 
-Builds for a few platforms are available as part of the RC here:
-https://repo.saltstack.com/salt_rc/. For RHEL and Ubuntu, Follow the
-instructions on https://repo.saltstack.com/, but prepend the URL paths with
-``salt_rc/``.
+Builds for a few platforms are available as part of the RC at:
+
+- https://repo.saltstack.com/salt_rc/
+- https://repo.saltstack.com/freebsd/salt_rc/
+
+.. note::
+
+    For RHEL and Ubuntu, Follow the instructions on
+    https://repo.saltstack.com/, but prepend the URL paths with ``salt_rc/``.
```

Comment:
Is `prepend` correct here? Or should it be `append`? I am not sure (especially given the example with the freebsd link above) what this link is supposed to look like. Can you give an example here of where the `salt_rc` portion is supposed to go in related to the RHEL or Ubuntu link?